lib/EvalSourceMapDevToolModuleTemplatePlugin.js

```diff
@@ -29,8 +31,10 @@ class EvalSourceMapDevToolModuleTemplatePlugin {
 		moduleTemplate.hooks.module.tap(
 			"EvalSourceMapDevToolModuleTemplatePlugin",
 			(source, module) => {
-				if (source.__EvalSourceMapDevToolData)
-					return source.__EvalSourceMapDevToolData;
+				if (cache.has(source)) {
+					return cache.get(source);
```

Comment:
I tried it a while ago and it seem to be more performant to just read from the cache and check for undefined after that, instead of doing two accesses to the WeakMap.

``` js
const cachedSource = cache.get(source);
if(cachedSource !== undefined) return cachedSource;
```

Reply:
Fixed